apps/analysis/models.py

```diff
@@ -388,6 +388,7 @@ def __str__(self):
 
 class AnalyticalStatement(UserResource):
     statement = models.TextField()
+    title = models.CharField(max_length=150, null=True, blank=True)
```

Comment:
Let's use this... same for TopicModelCluster as well. Maybe let's use `title` there as well.  Just for consistency.
```suggestion
    title = models.CharField(max_length=150, blank=True)
```